src/System.Net.Sockets.Legacy/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -8,6 +8,9 @@ namespace System.Net.Sockets.Tests
 {
     public class DualMode
     {
+        private const int DummyDualModeV6Issue = 8000;
```

Comment:
Nit: in the 8000 ranges I originally thought this is a port. Please add a TODO explaining that these are issues that need to be opened.

Also: if there's any sort of tooling scanning for the ActiveIssue string outside of the compiler, it will probably fail not understanding the Issue ID.